feat: トーク画面のアップデート

Imutaakihiro opened the pull request:
<!-- PRのbaseブランチはdevelopでお願いします！ -->

## 概要

<!-- このセクションでは、この PR の目的と概要を簡潔に説明してください。 -->

#9 トーク画面の作成と、追加時に自動でメッセージを作成

<!-- このセクションでは、この PR が関連する Issue やタスクをリンクしてください。以下のように記述します。 -->

- 関連 Issue: #123

## 変更点

<!-- このセクションでは、具体的な変更点や修正箇所をスクリーンショットや動画で示してください。 -->

- 変更点 1
動画
- 変更点 2
動画
- 変更点 3
動画

tomoimu20061209-creator committed:
[supabaseができなかった]

tomoimu20061209-creator committed:
[トーク画面の作成を渡す]

Commit:
feat: ハート認証時にグループチャットに通知メッセージを送信し、グレー表示で表示する機能を追加

- ハート認証時に「ハートが認証され、グループチャットに追加されました」というメッセージを送信
- 通知メッセージをグレー背景で中央揃え表示
- 通常のメッセージと区別できるようにスタイリング

tanaka-0224 commented on Iverse2:
@Imutaakihiro 
これは削除でお願いします！

tanaka-0224 commented on src/components/posts/PostBoardScreen.tsx:
```diff
@@ -102,17 +127,435 @@ export default function PostBoardScreen({ onNavigate }: PostBoardScreenProps) {
+        type: 'accepted'
+      });
+
+      const { data: notificationData, error: notificationError } = await supabase
```
@Imutaakihiro 
notificationっていうテーブルはないので、messageテーブルにis_systemというカラム追加しました！
これ使ってシステムの文章とユーザーの文章を分けてください！

`````
   const { data: notificationData, error: notificationError } = await supabase
        .from('message')
        .insert({
          user_id: user.id,
          board_id: likeRequest.board_id,
          is_system: true, // trueだとシステム、falseだとユーザー
          content: `${creatorName}さんが「${likeRequest.board.title}」への参加を承認しました`,
        })
        .select()
        .single();
`````

tanaka-0224 committed:
fix: messageテーブルにis_systemを追加

Commit:
chore: 不要なディレクトリ（Iverse, Iverse2）を削除

Commit:
feat: is_systemカラムを使用してシステムメッセージとユーザーメッセージを分離

- messageテーブルのis_systemカラムを使用してシステムメッセージを判定
- システムメッセージ（ハート認証通知、承認通知など）をグレー背景で中央揃え表示
- ハート認証時にis_system: trueでメッセージを送信
- notificationテーブルの参照をmessageテーブルに変更
- デバッグログを追加してis_systemの値を確認可能に

Imutaakihiro commented:
@tanaka-0224 
これでいけたと思います！
test1さんがtest6さんのハートを認証し、グループチャットに追加しました。

<img width="520" height="194" alt="image" src="https://github.com/user-attachments/assets/b75634aa-ad44-4ecf-a8b9-5b7f95b9cae1" />

Commit:
fix: ハート認証メッセージに承認者の名前を表示し、重複メッセージを削除

- 承認した人（ホスト）の名前もメッセージに含めるように修正
- メッセージテンプレートを「[APPROVER]さんが[USERNAME]さんのハートを認証し、グループチャットに追加しました。」に変更
- 重複していた承認通知メッセージを削除
- デバッグログを追加してメッセージ作成過程を確認可能に
